src/MPI.jl

```diff
@@ -58,6 +58,23 @@ end
 include("consts/consts.jl")
 using .Consts
 
+module API
+    using ..MPIPreferences
+    using ..Consts
+
+    names_Consts = filter(n -> startswith(string(n), "MPI_"), names(Consts; all = true)) |> collect
+    for n in names_Consts
+        @eval $n = Consts.$n  # signatures need types and constants
+    end
```

Comment:
what if we just moved all of `Consts` into `API` (and then just make `const Consts = API` for compatibility)

Reply:
If you wish.

Reply:
Mmmh it's less easier than thought.